src/periphery/FeeSplitter.sol

```diff
@@ -0,0 +1,112 @@
+pragma solidity ^0.8.18;
+
+import "lyra-utils/decimals/SignedDecimalMath.sol";
+
+import {Ownable2Step} from "openzeppelin/access/Ownable2Step.sol";
+import {IManager} from "v2-core/src/interfaces/IManager.sol";
+import {ISubAccounts, IAsset} from "v2-core/src/interfaces/ISubAccounts.sol";
+
+/**
+ * @title FeeSplitter
+ * @notice FeeSplitter is a contract that splits the balance of a subaccount held by this contract based on a % split
+ * @author Lyra
+ */
+contract FeeSplitter is Ownable2Step {
+  using SignedDecimalMath for int;
+
+  ISubAccounts public immutable subAccounts;
+  IAsset public immutable cashAsset;
+
+  uint public subAcc;
+  uint public splitPercent = 0.5e18;
+  uint public accountA;
+  uint public accountB;
+
+  constructor(ISubAccounts _subAccounts, IManager manager, IAsset _cashAsset) {
+    subAccounts = _subAccounts;
+    subAcc = _subAccounts.createAccount(address(this), manager);
+    cashAsset = _cashAsset;
+  }
+
+  ///////////
+  // Admin //
+  ///////////
+
+  /// @notice Set the % split
+  function setSplit(uint _splitPercent) external onlyOwner {
+    if (_splitPercent > 1e18) {
```

Comment:
I think cash and managers allow amount=0 transfers, but might be worth double checking. 

E.g. if `_splitPercent` is 0 or 1, you'd need to make 0 amount transfers in `split()`

Reply:
pushed test for this 👌